build/loc/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using CommandLine;
+
+// ReSharper disable SuggestVarOrType_BuiltInTypes
```

Comment:
nit remove these comments

Reply:
Removed comments. Used `SuppressMessage` attribute for all suppressions to be consistent.